index.bs

```diff
@@ -169,11 +207,71 @@ This allows developers to detect support for paint timing for a particular [=bro
 Processing model {#sec-processing-model}
 ========================================
 
+Recording paint timing {#sec-recording-paint-timing}
+--------------------------------------------------------
+
+<h4 id="sec-modifications-dom">Modifications to the DOM specification</h4>
```

Comment:
I don't think we want this to monkey-patch DOM or other specifications.

I think it's fine to define the "associated image request" concept here, and then populate it in the various image loading algorithms in HTML, given the fact that this supports LCP, which has cross-implementer support. It's also fine to do this in multiple stages (land the vague language now, while PRing hooks into the processing model)